persistence-modules/hibernate6/src/test/java/com/baeldung/attributevaluegenerator/WizardAttributeValueGeneratorIntegrationTest.java

```diff
@@ -0,0 +1,59 @@
+package com.baeldung.attributevaluegenerator;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.time.LocalDateTime;
+import java.util.UUID;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import net.bytebuddy.utility.RandomString;
+
+@SpringBootTest(classes = Application.class, properties = "spring.sql.init.data-locations")
+class WizardAttributeValueGeneratorIntegrationTest {
+
+    @Autowired
+    private WizardRepository wizardRepository;
+
+    @Test
+    void whenNewWizardRecordSaved_thenHouseGetsSorted() {
+        Wizard wizard = new Wizard();
+        wizard.setId(UUID.randomUUID());
+        wizard.setName(RandomString.make());
+
+        Wizard savedWizard = wizardRepository.save(wizard);
+
+        assertThat(savedWizard.getHouse()).isNotBlank();
+    }
+
+    @Test
+    void whenWizardRecordUpdated_thenUpdatedAtTimestampRefreshed() {
+        Wizard wizard = new Wizard();
+        wizard.setId(UUID.randomUUID());
+        wizard.setName(RandomString.make());
+
+        Wizard savedWizard = wizardRepository.save(wizard);
+        LocalDateTime initialUpdatedAtTimestamp = savedWizard.getUpdatedAt();
+
+        savedWizard.setName(RandomString.make());
+        Wizard updatedWizard = wizardRepository.save(savedWizard);
+
+        assertThat(updatedWizard.getUpdatedAt()).isAfter(initialUpdatedAtTimestamp);
+    }
+
+    @Test
+    void whenNewWizardRecordSaved_thenSpellPowerGenerated() {
+        Wizard wizard = new Wizard();
+        wizard.setId(UUID.randomUUID());
+        wizard.setName(RandomString.make());
+
+        Wizard savedWizard = wizardRepository.save(wizard);
+
+        assertThat(savedWizard.getSpellPower())
+          .isNotNull()
+          .isGreaterThan(50);
```

Comment:
In the PR we still use a 4 space indent, it's only 2-space in the article
```suggestion
        assertThat(savedWizard.getSpellPower())
            .isNotNull()
            .isGreaterThan(50);
```

You can use the automatic Baeldung formatter if you import the IDE formatter profile

Reply:
review incorporated.